regulations/templates/regulations/preamble-base.html

```diff
@@ -22,7 +22,11 @@
 {% block sub-head-class %}preamble-sub-head{% endblock %}
 
 {% block active_title %}
-  <em class="header-label">{{section_label}}</em>
+  <em class="header-label">
+    {% if section_label != None %}
```

Comment:
It may make more sense to put this logic in the Python view -- looks like the `None` is being set around line 481 of `regulations/views/preamble.py`. If not, we can at least make this check a wee bit more idiomatic with something like:
```
{% if section_label %}
```